src/components/elements/wysiwyg.tsx

```diff
@@ -73,8 +73,23 @@ const options: HTMLReactParserOptions = {
               nodeProps.className,
               "rs-pb-5 rs-pt-7 rs-px-1 border-l-3 border-archway-dark border-opacity-50"
             )
-            return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
           }
+          return <NodeName {...nodeProps}>{domToReact(children, options)}</NodeName>
+
+        case "table":
+          if (nodeProps.className && nodeProps.className.includes("sum-table")) {
+            nodeProps.className = twMerge(
+              nodeProps.className,
+              "centered 2xl:max-w-1200 relative left-1/2 w-screen -translate-x-1/2 break-words [&_tbody>tr>th]:bg-transparent [&_tfooter>tr]:bg-transparent [&_tr]:border-t-0 [&_td]:border-r-3 [&_td]:border-r-white [&_td]:rs-px-2 [&_td]:rs-py-1 [&_tbody>tr:nth-child(odd)>td:nth-child(1)]:bg-[#F8F7F6] [&_tfooter>tr:nth-child(odd)]:bg-[#F8F7F6] [&_tbody>tr:nth-child(odd)>td:nth-child(2)]:bg-spirited [&_tbody>tr>td:nth-child(2)]:text-white [&_tbody>tr:nth-child(odd)>td:nth-child(3)]:bg-poppy-dark [&_tbody>tr>td:nth-child(3)]:text-white [&_tbody>tr:nth-child(even)>td:nth-child(2)]:bg-[#F16C4B] [&_tbody>tr:nth-child(even)>td:nth-child(3)]:bg-poppy"
```

Comment:
looks like none of these color combinations meet the contrast ratio required for normal text. They are all below 3.7:1 with the lowest ratio being 2.7:1:
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=E04F39
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=D1660F
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F16C4B
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=E98300

Please work with designs to find compliant colors greater than 4.5:1

Reply:
New color set 😎 
https://webaim.org/resources/contrastchecker/?fcolor=2F2424&bcolor=F3795B
https://webaim.org/resources/contrastchecker/?fcolor=2F2424&bcolor=F6947C
https://webaim.org/resources/contrastchecker/?fcolor=2F2424&bcolor=F9A44A
https://webaim.org/resources/contrastchecker/?fcolor=2F2424&bcolor=FAB66E